src/Maestro/Maestro.Web/Api/v2019_01_16/Models/Build.cs

```diff
@@ -4,18 +4,14 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using JetBrains.Annotations;
 
 namespace Maestro.Web.Api.v2019_01_16.Models;
 
 public class Build
 {
     public Build([NotNull] Data.Models.Build other)
```

Comment:
Shouldn't the `NotNull` be removed here?

Reply:
Thanks, removed